### PR TITLE
Refactored 'test_integration.py' to use 'with open() as file' syntax

### PR DESCRIPTION
**Pull request by Rohan-cod**
## Description

Refactored the 'test_integration.py' file to use `with open() as file` syntax.

## How Has This Been Tested?

The code has been tested by running all the unit tests using the command `python -m unittest` and validated the PEP 8 formatting using `pycodestyle`.

- [x] Unit Tests
- [x] PEP 8 Formatting Validation

### CODE REVIEW

File: vcf2fhir/test/test_integration.py [hunk 1]
```diff
@@ -333,19 +341,21 @@ def test_multiple_region_studied(self):
...
+            # Validate the pased sequence relationship
+            self.assertEqual(_validate_phase_rel(
+                actual_fhir_json, {31: [24, 25], 32: [25, 26]}), True)
+            # Validate: list of observation uids and list of result uids same.
+            # Also set the uids to '' to avoid guid comparison in next step
+            map_ids = _get_uids_map(actual_fhir_json)
+            self.assertEqual(map_ids['obv_ids'], map_ids['result_ids'])
+            actual_fhir_json['issued'] = ''
+            # Finally, check if the acutal json after removing all uids is
+            # same as exppected json
```
Comment (srgothi92) [inline vcf2fhir/test/test_integration.py:356]: nit: typo from before. Fix in all other test too.
```suggestion
            # same as expected json
```

**Review by srgothi92 — COMMENTED** [review]
Great work 🍷 
 
 `with open()` is definitely right thing to use.  I am just curious, did you see any file close issues or you just realized, what existed before was not standard format and you should change it.

**Comment by Rohan-cod** [comment]
> Great work 🍷
> 
> `with open()` is definitely right thing to use. I am just curious, did you see any file close issues or you just realized, what existed before was not standard format and you should change it.

Yes @srgothi92, there were warnings as the files were not being closed after usage.
Screenshot 2021-04-08 at 9 10 24 PM

**Comment by Rohan-cod** [comment]
@srgothi92, I fixed the typos and squashed the changes into a single commit.

**Review by srgothi92 — APPROVED** [review]

**Pull request merged**